resource-aggregate/config.yaml

```diff
@@ -0,0 +1,85 @@
+log:
+  debug: false
+apis:
+  grpc:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+    authorization:
+      authority:
+      audience:
+      ownerClaim: sub
+      http:
+        maxIdleConns: 16
+        maxConnsPerHost: 32
+        maxIdleConnsPerHost: 16
+        idleConnTimeout: 30s
+        timeout: 10s
+        tls:
+          caPool: "/secrets/public/rootca.crt"
+          keyFile: "/secrets/private/cert.key"
+          certFile: "/secrets/public/cert.crt"
+          useSystemCAPool: false
+clients:
+  eventBus:
+    nats:
+      url:
+      tls:
+        caPool: "/secrets/public/rootca.crt"
+        keyFile: "/secrets/private/cert.key"
+        certFile: "/secrets/public/cert.crt"
+        useSystemCAPool: false
+  eventStore:
+    mongoDB:
+      uri:
+      database: eventStore
+      tls:
+        caPool: "/secrets/public/rootca.crt"
+        keyFile: "/secrets/private/cert.key"
+        certFile: "/secrets/public/cert.crt"
+        useSystemCAPool: false
+      # limits number of queries in one find request.
+      batchSize: 128
+      # limits number of connections.
+      maxPoolSize: 16
+      # close connection when idle time reach the value.
+      maxConnIdleTime: 4m0s
+    # tries to create the snapshot event after n events
+    snapshotThreshold: 16
+    # limits number of try to store event
+    occMaxRetry: 8
+  authorizationServer:
+    grpc:
+      address:
+      tls:
+        caPool: "/secrets/public/rootca.crt"
+        keyFile: "/secrets/private/cert.key"
+        certFile: "/secrets/public/cert.crt"
+        useSystemCAPool: false
+      keepAlive:
+        Time: 10s
+        Timeout: 20s
+        permitWithoutStream: true
+    pullFrequency: 15s
+    cacheExpiration: 1m
+    oauth:
+      clientID:
+      clientSecret:
+      scopes: []
+      tokenURL:
+      audience:
+      http:
+        maxIdleConns: 16
+        maxConnsPerHost: 32
+        maxIdleConnsPerHost: 16
+        idleConnTimeout: 30s
+        timeout: 10s
+        tls:
+          caPool: "/secrets/public/rootca.crt"
+          keyFile: "/secrets/private/cert.key"
+          certFile: "/secrets/public/cert.crt"
+          useSystemCAPool: false
+      tickFrequency: 10s
```

Comment:
again, describe purpose. Nobody knows what this configuration affects. Users just knows that it affects the execution of some function...

`verifyServiceTokenFrequency: 10s`

Reply:
fixed